src/adapters/picked_player_adapters.py

```diff
@@ -0,0 +1,56 @@
+from typing import List
+
+import prettytable as pt
+import requests  # type: ignore
+from core.picked_player import PickedPlayer
+
+f1_all_players = {}
+
+
+def player_id_to_name() -> dict:
+    global f1_all_players
```

Comment:
Usage of global variable to avoid HTTP call for each player

Reply:
To avoid this, you can pass the player list to the method. The HTTP call to get those players is made in the handler and then you pass it to the adapter method. So you would do a pure method and you are not having a global variable

Reply:
But, and you will hate this, you should use curried function :D